app/Main.hs

```diff
@@ -48,13 +49,19 @@ data Options = Options
   , oSecretFile      :: FilePath
   , oCmd             :: String
   , oArgs            :: [String]
-  , oConnectHttp     :: Bool
+  , oNoConnectTls    :: Bool
   , oNoValidateCerts :: Bool
-  , oInheritEnvOff   :: Bool
+  , oNoInheritEnv    :: Bool
   , oRetryBaseDelay  :: MilliSeconds
   , oRetryAttempts   :: Int
   } deriving (Eq, Show)
 
+data EnvSwitches = EnvSwitches
```

Comment:
Can you add a haddock string for this datatype that explains why we need this?
Esp. why we have both `EnvSwitches` and `EnvVar`.

(we should add haddock comments to everything at some point, but let's at least do so for new code for now)